lib/structuraid_core.rb

```diff
@@ -12,3 +12,5 @@ module StructuraidCore
 require_relative 'structuraid_core/materials/base'
 require_relative 'structuraid_core/engineering/base'
 require_relative 'structuraid_core/elements/base'
+
+require 'matrix'
```

Comment:
I think we should still keep loading these 3rd party dependencies on each file that requires it. That way it would be easier to understand where we are using them. What do you think?

Reply:
ok let's do it by that way